GSA/made-in-america
# [MIA-406] Urgent waiver tests and removal of old urgent waiver page

## Description

This pr adds test coverage to the new urgent waiver page... it is very similar to the non availability but has been updated to cover the new page 

Also, the removal of the MD file that held the old urgent waiver page has been removed.  This takes it out of the site map and keeps the sitemap tests from running on that page and a user from stumbling on it. 

## How to test

- pull branch down on local 
- run `npm run start`
- spilt terminal 
- run `npm run test` this will run all cypress tests headless 
- verify that all tests are passing 

for urgent page removal 
- run `npm run start`
- go to the browser and type `localhost:4000/sitemap.xml` and ensure that that `/about/urgent-waivers/` is not present in the sitemap 

## Additional information

This gets our test coverage back up to where it needs to be and keeps our site the most up to date 

To prepare for code review, please follow the following checks.
